_raml/apps/search/examples/post_search_products_response.json

```diff
@@ -0,0 +1,177 @@
+{
+    "results": 1,
+    "page": 1,
+    "resultsPerPage": 1,
+    "items": [
+      {
+        "productId": "54EF2E41-FCA8-E0E5-9323-0A0C05E66C2E",
+        "name": "Boom - Customised headphone",
+        "visible": true,
+        "productVariationType": "regular",
+        "productVariationMasterName": null,
+        "productVariationMasterId": null,
+        "manufacturerProductNumber": "CU001",
+        "productLength": null,
+        "productWidth": null,
+        "productHeight": null,
+        "productVariationSelection": null,
+        "shortDescription": null,
+        "deliveryPeriod": "2-4",
+        "description": "Customise your headphone in no time!",
+        "title": "Boom - Customized headphone - Technic365",
```

Comment:
Is it on purpose that you switch between en-GB and en-US spelling of customize in this PR? You also have a product called "Boom - Customised headphone" and the description of this product also uses "customise".

Reply:
👍